print-apps/oereb/general_info_and_disclaimer.jrxml

```diff
@@ -0,0 +1,65 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Created with Jaspersoft Studio version 6.17.0.final using JasperReports Library version 6.17.0-6d93193241dd8cc42629e188b94f9e0bc5722efd  -->
+<jasperReport xmlns="http://jasperreports.sourceforge.net/jasperreports" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://jasperreports.sourceforge.net/jasperreports http://jasperreports.sourceforge.net/xsd/jasperreport.xsd" name="topicmapboxlegend" pageWidth="252" pageHeight="155" whenNoDataType="AllSectionsNoDetail" columnWidth="252" leftMargin="0" rightMargin="0" topMargin="0" bottomMargin="0" uuid="6477891d-f5c5-456f-a115-e91b2a2768c6">
+	<property name="com.jaspersoft.studio.data.defaultdataadapter" value="One Empty Record"/>
+	<property name="com.jaspersoft.studio.unit." value="mm"/>
+	<style name="Default" isDefault="true" fontName="Cadastra" fontSize="8"/>
+	<parameter name="GeneralInformation" class="java.lang.String"/>
+	<parameter name="GeneralInformationLabel" class="java.lang.String"/>
+	<parameter name="DisclaimerGrundbuch_Title" class="java.lang.String"/>
+	<parameter name="DisclaimerGrundbuch_Content" class="java.lang.String"/>
+	<parameter name="SUBREPORT_DIR" class="java.lang.String"/>
+	<title>
+		<band height="47">
+			<textField textAdjust="StretchHeight" isBlankWhenNull="true">
+				<reportElement positionType="Float" stretchType="ContainerBottom" x="0" y="11" width="230" height="36" isPrintWhenDetailOverflows="true" uuid="d8faf59a-0317-4cf0-9ffd-49759fee539e">
+					<property name="com.jaspersoft.studio.unit.y" value="px"/>
+				</reportElement>
+				<textElement>
+					<font fontName="Cadastra" size="6"/>
+					<paragraph lineSpacingSize="11.0"/>
+				</textElement>
+				<textFieldExpression><![CDATA[$P{GeneralInformation}]]></textFieldExpression>
+			</textField>
+			<textField isBlankWhenNull="false">
+				<reportElement x="0" y="0" width="230" height="11" isPrintWhenDetailOverflows="true" uuid="361fca0e-48b8-4d23-863d-4cdb82c500a2">
+					<property name="com.jaspersoft.studio.unit.width" value="pixel"/>
+					<property name="com.jaspersoft.studio.unit.height" value="mm"/>
+					<property name="com.jaspersoft.studio.unit.x" value="pixel"/>
+					<property name="com.jaspersoft.studio.unit.y" value="mm"/>
+				</reportElement>
+				<textElement markup="styled">
+					<font fontName="Cadastra" size="6" isBold="true"/>
+					<paragraph lineSpacingSize="8.0"/>
+				</textElement>
+				<textFieldExpression><![CDATA[$P{GeneralInformationLabel}]]></textFieldExpression>
+			</textField>
+		</band>
+	</title>
```

Comment:
I think I would have used detail bands here. But if title is working I would not touch it ;-)
If something works in the Print templates it is best to not touch it :-D